02.calendar/calendar.rb

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env ruby
+require 'date'
+require 'optparse'
+
+# コマンドラインオプション設定(デフォルト値は今日の年と月)
+params =ARGV.getopts("m:", "y:", "m:#{Date.today.mon}", "y:#{Date.today.year}")
+input_month = params["m"].to_i
+input_year = params["y"].to_i
+
+# 月の最初の日にちを取得
+start_date = Date.new(input_year, input_month, 1)
+
+# 月の最終日を取得
+end_date = Date.new(start_date.year, start_date.mon, -1)
```

Comment:
```suggestion
end_date = Date.new(input_year, input_month, -1)
```

でも同じになるはず。
同じ値なら同じ変数から取得した方が「同じ値だよね」というのがコードからぱっと把握できてベターです

Reply:
修正しました！
c379c229112fc337d47d52738bf85791ddedc9c3